thumbor/thumbor
# Fix meta width and height when exif orientation is not default and RESPECT_ORIENTATION is true

## What does this PR do?
Fix #949 

## How do I test this PR?
This PR accompanies an image with exif orientation="Rotate 90 CW". Try to get meta information with thumbor and check if width and height  match with the image rotated.
